WeScan/Review/ReviewViewController.swift

```diff
@@ -15,6 +16,15 @@ final class ReviewViewController: UIViewController {
     private var enhancedImageIsAvailable = false
     private var isCurrentlyDisplayingEnhancedImage = false
     
+    lazy var imageCheck: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = UIImage(named: "img_check")
+        imageView.backgroundColor = UIColor(red: 1, green: 1, blue: 1, alpha: 0.0)
```

Comment:
To keep dark mode in mind, could we use a semantic color instead? Or if a specific color is required, use a named version?
```suggestion
        imageView.backgroundColor = .black
```